src/sites/amsterdam/components/contact/Bedankt.tsx

```diff
@@ -0,0 +1,28 @@
+import { Breadcrumb, Column, Grid, Heading, Paragraph } from '@amsterdam/design-system-react'
+import { siteUrl } from '../../constants'
+import { Link } from '../../../../shared/components/Link'
+
+export const Bedankt = () => (
+  <Grid paddingBottom="medium">
+    <Grid.Cell span={{ narrow: 4, medium: 5, wide: 8 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
```

Comment:
We can span this cell over all columns in the medium grid; also for the other form pages please.

```suggestion
    <Grid.Cell span={{ narrow: 4, medium: 8, wide: 8 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
```

Reply:
Done